README.rst

```diff
@@ -76,9 +76,6 @@ If you have more than one role available to you, you'll be prompted to
 choose the role from a list; otherwise, if your credentials are correct,
 you'll just see the AWS keys printed on stdout.
 
-You should ``eval`` the ``export`` statements that come out, because
```

Comment:
How about rewriting the `Usage` section of the README. Something along the lines of:

* Usage
    * Configuration via environment variables (setting GOOGLE_USERNAME etc)
    * Writing to a profile configuration (which arguments to use)
    * Using credentials in environment variables only (leaving off profile argument)
    * Running from a Docker image
    * Running from a locally-installed Python package

and then, I can merge this into the bash_wrapper branch and update the section on "environment variables only"